src/vector/platform/WebPlatform.ts

```diff
@@ -184,17 +183,8 @@ export default class WebPlatform extends VectorBasePlatform {
 
     getDefaultDeviceDisplayName(): string {
         // strip query-string and fragment from uri
-        const u = url.parse(window.location.href);
-        u.protocol = "";
-        u.search = "";
-        u.hash = "";
-        // Remove trailing slash if present
-        u.pathname = u.pathname.replace(/\/$/, "");
```

Comment:
So it does look like the behaviour here has been lost.

Previously it'd pass through things like `riot.im/app` happily too but I think now that'll instead just be `riot.im`

Reply:
Hmm... will check that out soon

Reply:
Yep it can be fixed with this (commiting this approach now)
![image](https://user-images.githubusercontent.com/4066435/107661608-7ba36000-6c9a-11eb-90f7-1ecc950625a6.png)